CONTRIBUTING.md

```diff
@@ -18,4 +18,54 @@ After you have contributed something check your code by using:
 $ npm run linting | npm test
 ```
 
+## Debugging
+
+Included in the modules is some basic debugging logs. You can set each of these individually to target the log of interest:
+
+```js
+var diacritics = require("diacritics-transliterator");
+diacritics.debug = {
+    server: true, // show server & cache interactions
+    regexp: true, // show resulting regular expressions
+    regexpTests: true, // show all regular expression tests
+    placeholder: true // show string breakdown and results
+};
+diacritics.getVariant("de");
+// logged as: "http://api.diacritics.io/v1/?language=de; response: loaded"
+diacritics.getVariant("de");
+// logged as: "http://api.diacritics.io/v1/?language=de; {loaded from cache}"
+```
+
```

Comment:
I added the debugging while I was adding tests because I wanted to see why some tests were failing. I don't think it would be necessary to have the output as a user, so I didn't add it as an option to each method.

The `debug.regexpTests` setting was originally included with the `debug.regexp`. When set to `true`, all you would see was screen after screen of logs, so I split it to two parts. This setting isn't really necessary. I can remove it in the final version.